Library/Formula/crystax-ndk.rb

```diff
@@ -2,14 +2,14 @@ class CrystaxNdk < Formula
   desc "Drop-in replacement for Google's Android NDK"
   homepage "https://www.crystax.net/android/ndk"
 
-  version "10.2.1"
+  version "10.3.0"
 
   if MacOS.prefer_64_bit?
-    url "https://www.crystax.net/download/crystax-ndk-#{version}-darwin-x86_64.7z"
-    sha256 "1503eec2b883ffbe8f24bcfd2f3d47579ff1c9ce84be3612d8cfe5339aa0df40"
+    url "https://www.crystax.net/download/crystax-ndk-#{version}-darwin-x86_64.tar.xz"
```

Comment:
CrystaX NDK 10.3.0 don't provide 7z archives anymore, only tar.xz. You can check it on https://www.crystax.net/en/download.